tests/libcxx/tests.supported

```diff
@@ -2454,7 +2454,6 @@
 ../../3rdparty/libcxx/libcxx/test/std/language.support/support.types/byteops/xor.pass.cpp
 ../../3rdparty/libcxx/libcxx/test/std/language.support/support.types/max_align_t.pass.cpp
 ../../3rdparty/libcxx/libcxx/test/std/language.support/support.types/null.pass.cpp
-../../3rdparty/libcxx/libcxx/test/std/language.support/support.types/nullptr_t.pass.cpp
```

Comment:
Why is this no longer supported?

Reply:
@asvrada could you answer this one please?

Reply:
This fails after switched to clang-11, disable for now.